BridgeSDK/BridgeAPI/SBBUploadManager.m

```diff
@@ -430,18 +430,24 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
     NSMutableDictionary *headers = [NSMutableDictionary dictionary];
     [self.authManager addAuthHeaderToHeaders:headers];
     [self.networkManager post:ref headers:headers parameters:nil completion:^(NSURLSessionDataTask *task, id responseObject, NSError *error) {
+
 #if DEBUG
-      if (error) {
-        NSLog(@"Error calling upload complete for upload ID %@:\n%@", uploadSession.id, error);
-      } else {
-        NSString* uploadStatusUrlString = nil;
-        if ([self.networkManager isKindOfClass:[SBBNetworkManager class]]) {
-          NSString* relativeUrl = [NSString stringWithFormat:@"/api/v1/upload/%@/status", uploadSession.id];
-          NSURL* url = [(SBBNetworkManager*) self.networkManager URLForRelativeorAbsoluteURLString:relativeUrl];
-          uploadStatusUrlString = [url absoluteString];
+        if (error) {
+            NSLog(@"Error calling upload complete for upload ID %@:\n%@", uploadSession.id, error);
+        } else {
+            NSString* uploadStatusUrlString = nil;
+            if ([self.networkManager isKindOfClass:[SBBNetworkManager class]]) {
+                NSString* relativeUrl = [NSString stringWithFormat:@"/api/v1/upload/%@/status", uploadSession.id];
+                NSURL* url = [(SBBNetworkManager*) self.networkManager URLForRelativeorAbsoluteURLString:relativeUrl];
+                uploadStatusUrlString = [url absoluteString];
+                
+                if ([_uploadDelegate respondsToSelector:@selector(uploadManager:uploadOfFile:completedWithVerificationURL:)]) {
+                    
```

Comment:
The extraneous blank lines don't follow the whitespace conventions of the rest of the file.

Also, maybe it'd be good to fix the tab-spacing of the whole file to be consistent with this; I originally developed this on my personal laptop which I'd set for a previous job to two-space tabs, but have since standardized on four for new files, as you've used here.